rancher2/resource_rancher2_cluster_v2.go

```diff
@@ -64,6 +70,37 @@ func resourceRancher2ClusterV2() *schema.Resource {
 	}
 }
 
+func resourceRancher2ClusterV2Resource() *schema.Resource {
+	return &schema.Resource{
+		Schema: clusterV2FieldsV0(),
+	}
+}
+
+func resourceRancher2ClusterV2StateV0(rawState map[string]any, meta interface{}) (map[string]any, error) {
+	if rkeConfigs, ok := rawState["rke_config"].([]any); ok && len(rkeConfigs) > 0 {
+		for i := range rkeConfigs {
```

Comment:
Is there another place that checks if the passed struct is ok?  This if it isn't this function will just ignore it.